scripts/eslint-rules/validate-comment-tags.cjs

```diff
@@ -0,0 +1,39 @@
+// eslint-rules/validate-comment-tags.js
+
+module.exports = {
+    meta: {
+      type: "problem",
+      docs: {
+        description: "Ensure TODO, FIXME, DP tags follow the specific format",
+        category: "Best Practices",
+        recommended: false
+      },
+      fixable: "code",
+      schema: [] // no options
+    },
+    create(context) {
+      const contains_tag = /(TODO|FIXME|DP)/;
+      const correct_syntax = /^[\s|*]+(TODO|FIXME|DP): @\w+: \d{2}-\d{2}-\d{4} : \d+ : .+ : .+/;
```

Comment:
we should maintain a map for tag vs syntax as DP and TODO can have different syntax